packages/gatsby-theme-styleguide/src/gatsby-components/Preview/Preview.js

```diff
@@ -17,13 +17,29 @@ class Preview extends PureComponent {
     }
   }
 
+  getPreviewPath = () => {
+    // src/docs/Components/Buttons/previews/button-styles
+    const basePath = 'src/docs'
+    const { location, path } = this.props
+
+    // we need only pieces of path
+    let locationParsed = location.pathname.replace(/^\/|\/$/g, '').split('/')
```

Comment:
woking on windows ?

Reply:
path.sep() (from node API) is exactly for that and support windows paths

Reply:
No, in this case we're only dealing with a string, this parsed string will be used to make a comparison. Anyway, It was tested on windows 10.